client/src/components/ArtDetails.jsx

```diff
@@ -15,19 +18,22 @@ function ArtDetails({ art, onClose }) {
         </button>
       </header>
       <article className="modal-content-artDetails">
-        <img
-          src={art.image}
-          alt={art.title}
-          className="modal-picture-artDetails"
-        />
-
+        {art.image ? (
+          <picture className="modal-picture-artDetails">
+            <img src={imagePath} alt={art.title} />
+          </picture>
+        ) : (
+          <p>Aucune image disponible</p>
+        )}
         <h2>
-          {art.title}
+          {art.title ? art.title.toUpperCase() : "TITRE"}
           {art.title && art.artist && " - "}
-          {art.artist}
+          {art.artist ? ` - ${art.artist}` : null}
         </h2>
-        <p className={art.description && "modal-artDetails-description"}>
-          {art.description}
+        <p>
+          {art.information
+            ? art.information
+            : "Aucune description n'est disponible pour le moment."}
```

Comment:
After our review from dev to main, I changed ArtDetails so that there are no information appearing in the modal when there is nothing provided from the database, it is also what I told you on Discord. The changes were merged into dev no later than this week. So please come back to the version in dev.

Reply:
👍